midi_tokenizers/base_tokenizers/exponential_time_tokenizer.py

```diff
@@ -143,6 +143,7 @@ def quantize_frame(self, df: pd.DataFrame):
         df["velocity_bin"] = np.digitize(df["velocity"], self.velocity_bin_edges) - 1
         df["start"] = np.round(df["start"] / self.min_time_unit) * self.min_time_unit
         df["end"] = np.round(df["end"] / self.min_time_unit) * self.min_time_unit
+        df.loc[df["start"] == df["end"], "end"] += self.min_time_unit
```

Comment:
```suggestion
        # We have to manually prevent notes with 0.0 duration after rounding
        df.loc[df["start"] == df["end"], "end"] += self.min_time_unit
```

Comments are great for non-obvious steps :)